test/ci-test-weather.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
In other words, my alarm here is that we are not testing custom complex code, just static declarative files, so tests have no reason to be manually developed instead of automatically generated.